File: baekjoon/java/10818. 최소, 최대.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int max, min;
        int[] arr = new int[n];
        arr[0] = sc.nextInt();
        min = max = arr[0];
        for (int i=1;i<n;i++){
            arr[i] = sc.nextInt();
            if (arr[i] < min) min = arr[i];
            else if (arr[i] > max) max = arr[i];
        }
        System.out.println(min+" "+max);
    }
}

File: baekjoon/cpp/10818. 최소, 최대.cpp
#include <iostream>
using namespace std;


int main(){

    int n, min, max;
    cin >> n;
    int arr[n];
    
    for (int i = 0; i < n;i++){
        cin >> arr[i];
        if (i==0) min = max = arr[0];
        else if (max < arr[i]) max = arr[i];
        else if (min > arr[i]) min = arr[i];
        
    }
    cout << min << " " << max;


    return 0;
}
